templates/default/event/eventinfoshort.html

```diff
@@ -208,7 +208,7 @@
         </TD>
         <TD class="nobr">
         {if $event.netnode_name}
-            {icon name="netnode"}<a href="?m=netnodeinfo&id={$event.netnode_id}">{$event.netnode_name|escape}</a>
+            {button type='link' icon='netnode' href="?m=netnodeinfo&id={$event.netnodeid}" label=$event.netnode_name}
```

Comment:
Czy tutaj netnode_id zmieniasz się na netnodeid w ramach jakiejś poprawki, czy po prostu również gdzieś w backendzie to się znienia?

Reply:
`netnode_id` nie działa, zaproponowany stan działa.